alarm/binance_utils.py

```diff
@@ -7,15 +7,32 @@
 logger = logging.getLogger(f'{__name__}')
 
 
-def get_binance_trade_pairs():
+def get_binance_exchange_info():
     binance_exchange_info_url = "https://api.binance.com/api/v3/exchangeInfo"
     response = requests.get(binance_exchange_info_url)
+    if response.status_code != 200:
+        raise requests.exceptions.RequestException("API request for get binance exchange data was failed")
 
-    if response.status_code == 200:
-        valid_list_of_coin_names = [symbol['symbol'].lower() for symbol in response.json()['symbols']]
-        return valid_list_of_coin_names
-    else:
-        return []
+    return response.json()
+
+
+def format_trade_pair_for_message(trade_pair):
```

Comment:
So why do we need this function? It says we want to format a trade pair. But what it actually does? It validates, that trade pair is a valid Binance trade pair. So the function name is not appropriate to what it does.

Reply:
Without a match check, we can't do the formatting, because we pull ['baseAsset'] and ['quoteAsset'] from response.json() and return the formatted string. Correct me if I think it's wrong, but if is_valid_binance_trade_pair function returns true, then in create_message_about_threshold_break we should also do a match check to use ['baseAsset'] and ['quoteAsset'] and format the string, and this is already duplicated code or am I not right?